Add UTC as an option

jniechcial opened the pull request:
Selected date via Pikaday is returned in local timezone. I bumped into situation where everywhere in application we parse dates to UTC format as it's used across variety of timezones. It made an issue that dates selected on additive time zones were parsed as yesterday (subtracting hours from 00:00 at selected date).

I propose a simple solution to pass `utc` option to ember-pikaday that will take care to format the date to UTC before setting in on `value` property. I provide test for that feature in advance.

The feature uses `momentjs`, but as it is already in `bower` I do not see any disadvantages of such solution.

stravid commented:
Thanks a lot @jniechcial this looks really useful. Using Moment.js is no problem at all, it is explicitly mentioned and encouraged in the readme (regarding the format of the input element).

I find time in combination with timezones and formats really confusing so I will think this all through and look for a way to communicate this new feature clearly in the readme.

Do you think we have to take care of the case where the datepicker value is set by the application? Is it possible that the datepicker shows a wrong date in this case?

jniechcial commented:
I tested two cases:
* when the date is set using `#set` on `value` field directlty with non-UTC time-zone, the component shows proper date, but does not convert it automatically to UTC format (which results in the same situation I bumped into in description - yesterday's date after conversion the date that is `#get` from the `value`). However, I believe that it is proper behaviour as the component should not convert the objects that are passed to it directly and (maybe on purpouse) have other time zone than UTC.
* when the date is set by the user using `input` field, the component shows proper date and returns in value properly converted date to UTC.

stravid commented:
To keep you updated, I'm not letting you hang, I'm just unsure how to handle the case when the datepicker is set via `get`.

jniechcial commented:
Thanks, I understand. However, you meant `set`, don't you? 

I see only two possible approaches - one that converts the value to UTC and one that don't. Hard to say what will be better as both have pros and cons. I think that the most important thing after chosing one of them would still be to clearly communicate the behaviour in `README` as both of them may be clear to some people and unclear to others.

stravid commented:
Yes indeed, I was obviously talking about `set`, my mistake!

stravid commented:
Sorry for having you wait so long. I discussed the issue with my colleague and we made a decision. Let's keep it the way you have implemented it, when the value is set by the application the datepicker will just take it and use it as it is. We can assume that the application takes care of things if it wants to use UTC.

I made a few comments regarding the code and the readme needs an update, once that's done it is good to go. Thanks again for your work and patience!

stravid commented on addon/components/pikaday-input.js:
```diff
@@ -35,7 +37,12 @@ export default Ember.Component.extend({
   userSelectedDate: function() {
-    this.set('value', this.get('pikaday').getDate());
+    var selectedDate = this.get('pikaday').getDate();
+    if (this.get('utc'))
```
I would like to name the attribute `useUTC` since it makes it more obvious what it does. In addition the `{` needs to be on the same line as the `if`.

jniechcial commented:
Thanks for your reply. I fixed the code according to your comments and added README about that option. However, I did not change `CHANGELOG`. I assume you will do it in right time after next release.

stravid commented:
Thanks for your work @jniechcial I just released a new version with your feature!
